Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Themes/InteractableStringTheme.cs

```diff
@@ -13,12 +13,13 @@ namespace Microsoft.MixedReality.Toolkit.UI
     /// </summary>
     public class InteractableStringTheme : InteractableThemeBase
     {
+        private TMPro.TextMeshPro meshPro;
         private TextMesh mesh;
         private Text text;
 
         public InteractableStringTheme()
         {
-            Types = new Type[] { typeof(TextMesh), typeof(Text) };
+            Types = new Type[] { typeof(TMPro.TextMeshPro), typeof(TextMesh), typeof(Text) };
```

Comment:
Please add support for TextMeshProUGUI as well to cover all the bases. Thanks for this BTW.